tests/docker_images/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile that installs imap-data-access and its dependencies
+FROM public.ecr.aws/docker/library/python:3.10-slim
+USER root
+
+# Upgrade pip to the latest version
+RUN pip install --upgrade pip
+
+RUN pip install requests
```

Comment:
We shouldn't need either of these now.
```suggestion
```

Reply:
Even though this code doesn’t use it, the imap-data-access uses it. Will it fail on that? I will check that later too

Reply:
`imap-data-access` shouldn't use requests. If it did the `pip install xyz` should pick that up and install the required dependencies as well.